packages/web-config-server/src/apiV1/utils/findAccessibleGroupedMapOverlays.js

```diff
@@ -158,14 +158,25 @@ const sortMapOverlayItems = (mapOverlayItems, relations) => {
 const translateOverlaysForResponse = mapOverlays =>
   mapOverlays
     .filter(({ presentationOptions: { hideFromMenu } }) => !hideFromMenu)
-    .map(({ id, name, presentationOptions, report_code: reportCode, dataElementCode, legacy }) => ({
-      id, // just for sorting purpose, will be removed later
-      name,
-      ...presentationOptions,
-      reportCode,
-      dataElementCode,
-      legacy,
-    }));
+    .map(
+      ({
+        id,
+        name,
+        linkedMeasures,
+        presentationOptions,
+        report_code: reportCode,
+        dataElementCode,
+        legacy,
+      }) => ({
+        id, // just for sorting purpose, will be removed later
+        measureIds: [id, ...(linkedMeasures || [])],
```

Comment:
Need to add back `measureIds` in `measures` end points to work with `displayedMapOverlay` state and switch buttons.